### DISCUSSION TITLE
Extension packs

### DISCUSSION THREAD

**Discussion by C1710** [C1710/blobmoji#80] [Ideas]
I'm already working on an emoji_builder update (I really need to find a nice name for it...) that allows to bundle emojis (and tables) into _packs_.  
This would not only allow to just use Noto emojis (or Twemoji or whatever) for all missing emojis when I'm again very late with a release, but it would also allow to add stuff that is controversial/not for everyone.

Some examples:
- Legacy/"Classic" Blobs like in #66
- Pride/rainbow emojis (like the rainbow colored train, football, hearts, etc.)
- A real xor water pistol
- Emojis that do not exist yet/ligatures that don't require a ZWJ, but would cause confusion in general
  - For example 😱 + 🥳 → Excited face dot svg - in this case _I_ would know where this comes from and I could also understand what is actually meant by that emoji if it was falsely displayed
- Versions could also be separated, instead of the mess that the svg-folder is right now

Any more ideas?

**Comment by ghost** [comment]
Is the pack done yet?

**Reply by C1710** [reply]
No, there are none yet

**Reply by ghost** [reply]
Why not?

**Reply by thayerfox** [reply]
@MarcoMahone721 because he doesn't work for you?  Its a hobby project, let him work at his own pace.

**Reply by Bronnel721** [reply]
@thayerfox, to be honest I don't really mind that he didn't make them yet. This is the same person, but toned down.
